docs/zz_generated.seed.yaml

```diff
@@ -367,6 +367,12 @@ spec:
     envoy:
       # DockerRepository is the repository containing the component's image.
       dockerRepository: docker.io/envoyproxy/envoy-alpine
+      loadBalancerService:
+        # Annotations are used to further tweak the LoadBalancer integration with the
+        # cloud provider.
+        annotations: null
+        # LoadBalancerSourceRanges will restrict front-loadbalancer service to specific ip ranges
+        sourceRanges: null
```

Comment:
Just from looking at this, I would not know how to configure this. Is Source ranges an object? A map? Does it look like AWS EC2 ingress rules? Are those CIDRs? Can you add some sentence to the docs like "This should be a list of ............." and ideally extend the example YAML generator so it doesn't generate `null`, but `[]` (which is a bit more helpful also).

Same goes for the annotations field, which would look nicer as `[]` in the example YAML.